.tours/-welcome-to-e-bikes.tour

```diff
@@ -0,0 +1,72 @@
+{
+  "$schema": "https://aka.ms/codetour-schema",
+  "title": "👋 Welcome to E-Bikes",
+  "steps": [
+    {
+      "file": "force-app/main/default/lwc/productTileList/productTileList.js",
+      "description": "The Product Explorer lets you browse thought E-Bikes products by displaying cards that contain the product and it's price. It gets these by using an @wire to query call the getProducts Apex Class and providing the filter values that have been set. \n",
+      "line": 48
+    },
+    {
+      "file": "force-app/main/default/lwc/productCard/productCard.js",
+      "description": "The product card subscribes to the PRODUCT_SELECTED_MESSAGE message channel and passes the payload to the handleRecordLoaded method which will query for and return the selected record.",
+      "line": 57,
+      "selection": {
+        "start": {
+          "line": 57,
+          "character": 1
+        },
+        "end": {
+          "line": 57,
+          "character": 55
+        }
+      }
+    },
+    {
+      "file": "force-app/main/default/lwc/similarProducts/similarProducts.js",
+      "description": "The Similar Products component is accessed by a user on a record detail page. In LWC you can use the `@api recordId` property to gather the id from the pages context. ",
+      "line": 9
+    },
+    {
+      "file": "force-app/main/default/lwc/similarProducts/similarProducts.js",
+      "description": "Using Lightning Data Services, we can use an `@wire` to query for a records fields by passing in the `recordId` and the fields property that we have defined.",
+      "line": 15,
+      "selection": {
+        "start": {
+          "line": 6,
+          "character": 1
+        },
+        "end": {
+          "line": 7,
+          "character": 1
+        }
+      }
+    },
+    {
+      "file": "force-app/main/default/lwc/similarProducts/similarProducts.html",
+      "description": "When the similarProducts are returned from the Lightning Data Service transaction, we can iterate over the result and pass the data into the Product List Item component.",
+      "line": 5
+    },
+    {
+      "file": "force-app/main/default/lwc/productListItem/productListItem.js",
+      "description": "Once the values have been populated by the parent component, in this case similarProducts, you can select an item to view it's details. In this case, we are using Lightning Navigation Service to navigate to a record page and include the `recordId` and `objectApiName` to ensure that the user gets to the correct record.",
+      "line": 13
+    },
+    {
+      "file": "force-app/main/default/lwc/productTileList/productTileList.js",
+      "description": "The LWC uses Lightning Message Service to publish a selected product using the PRODUCT_SELECTED_MESSAGE message channel. This way, any components subscribing to the message will recieve the data when it is published.",
```

Comment:
- This steps come too late because you start talking about LMS in step 2. You may want to reorder these two steps.
- Add code tags around `PRODUCT_SELECTED_MESSAGE`.